packages/internal/bridge/bridge-sample-app/src/lib/helpers.ts

```diff
@@ -0,0 +1,78 @@
+import { CHILD_ERC20, CHILD_ERC20_BRIDGE, ERC20, ROOT_ERC20_BRIDGE_FLOW_RATE } from "@imtbl/bridge-sdk";
+import { providers, ethers } from "ethers";
+
+export function delay(time: number) {
+    return new Promise(resolve => setTimeout(resolve, time));
+}
+
+export async function waitForReceipt(txHash: string, provider: providers.JsonRpcProvider) {
```

Comment:
out of curiosity, was it not possible to use the ethers [`waitForTransaction()` function](https://docs.ethers.org/v5/api/providers/provider/#Provider-waitForTransaction)

Reply:
I pulled these helpers from Zhenyang's e2e scripts, so i just used what he had already done. We could change it but its probably not a lot of difference.